OOP/oop_home_task.py

```diff
@@ -0,0 +1,475 @@
+"""
+Home task for Module 5 - Classes. OOP.
+
+Create a tool, which will do user generated news feed:
+
+1.User select what data type he wants to add
+
+2.Provide record type required data
+
+3.Record is published on text file in special format
+
+
+
+You need to implement:
+
+1.News – text and city as input. Date is calculated during publishing.
+
+2.Private ad – text and expiration date as input. Day left is calculated during publishing.
+
+3.Your unique one with unique publish rules.
+
+
+
+Each new record should be added to the end of file. Commit file in git for review.
+"""
+
+import datetime
+
+SEPARATOR = '------------------------------\n\n'
+
+
+class Publication:
+    """
+    Class represents publication entity.
+    """
+    def __init__(self, publication_date, publication_type, city, publication_text):
+        """
+        Function is constructor for Publication.
+        @param publication_date: date of publication
+        @param publication_type: type of publication
+        @param city: city where something happened
+        @param publication_text: text of publication
+        """
+        self.publication_date = publication_date
+        self.publication_type = publication_type
+        self.city = city
+        self.publication_text = publication_text
+
+
+class UserInterface:
+    """
+    Class for interaction with user inputs.
+    """
+    @staticmethod
+    def ask_city():
+        """
+        Method for entering of city for publication.
+        @return: city entered by user
+        """
```

Comment:
The same for docu, it's obvious what this function does.

Reply:
I will not place any comments about docu anywhere anymore, just play with it yourself

Reply:
Fixed.